src/main/java/org/humancellatlas/ingest/core/exception/MultipleOpenSubmissions.java

```diff
@@ -0,0 +1,28 @@
+package org.humancellatlas.ingest.core.exception;
+
+public class MultipleOpenSubmissions extends RuntimeException {
```

Comment:
please append "Exception" to the class name to make it clear it is an exception class.

Reply:
Renamed. Thanks!